01-Introduction and Goals.md

```diff
@@ -4,4 +4,6 @@
 
 ## Quality Goals
 
-## Stakeholders
\ No newline at end of file
+* Entity data which is included in a client wiki's article content is eventually consistent with updates on the corresponding repo within XYZ minutes
```

Comment:
Do we actually have a number for the maximum acceptable lag time? I heard "a few minutes" mentioned several times but a concrete value would make this a better quality goal.

Reply:
We don't for this part (on the client side), but we do on the repo side (and this is the only side that we currently actively look at and measure).
The repo side is our "dispatching" system, and the client side is a couple of jobs that end up in queues.

The withins X minutes bit kind of ends up being a deployment detail?
If you deploy this without a parser cache and without a web cache it will always be instantly(ish) consistent?
Not sure how to best reflect that in our docs, but perhaps these minutes need to live somewhere else?

Reply:
Right, I think I agree that this section shouldn't go into too much detail on this. How about phrasing it in a more fluffy way, something like:
```suggestion
* Entity data which is included in a client wiki's article content is eventually consistent with updates on the corresponding repo. The updates from the repo are available to the client within XYZ minutes.
```

That would put the non-functional requirement fully on the repo side, and leave the details on how the data gets into the article to the client, which I think makes sense.

Reply:
Looks good to me as a version one (I imagine we will keep tweaking these things as we move forward rather than making them perfect now.